src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -63,11 +63,8 @@ export interface CodeActionShowOptions {
 	readonly fromLightbulb?: boolean;
 }
 export interface ICodeActionMenuItem {
-	title: string;
-	detail: string;
 	action: IAction;
-	decoratorRight?: string;
-	isSeparator?: boolean;
+	isSeparator: boolean;
```

Comment:
Keeping isEnabled, isDocumentation in the menuItem because it's checked as a list widget option/item elsewhere in the code.